listener/upload.go

```diff
@@ -50,6 +51,7 @@ const (
 	RepoPathPattern           = "(/content/.*)"
 	RepoBasearchPlaceholder   = "$basearch"
 	RepoReleaseverPlaceholder = "$releasever"
+	TemplateRepoPattern       = `^https://cert\.console.*/api/pulp-content/[[:xdigit:]]+/templates/`
```

Comment:
newer domains will have cs- prepended to the 'domain' name so you might need something like:    `(cs-)?[[:xdigit:]]+`

Reply:
👍🏼